### PR TITLE
Login logout 13

### PR DESCRIPTION
**Pull request by muydanny**
User story 13 pt 1 - regular user

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+  def create
+    user = User.find_by(email: params[:email])
+    
+    if user.authenticate(params[:password])
+      session[:user_id] = user.id
+      flash[:success] = "Welcome, #{user.name}! You are now logged in."
+      redirect_to '/'
+    else
+      flash[:error] = "Sorry, youre not one of us."
```
Comment (stellakunzang) [inline app/controllers/sessions_controller.rb:14]: * you're ❤️

Comment (sagemlee) [inline app/controllers/sessions_controller.rb:14]: cold.

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      render :new
+    end
+  end
+end
```
Comment (Gallup93) [inline app/controllers/sessions_controller.rb:18]: Good work! I think I'm going to need this for my user stories as well

File: app/views/welcome/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<h1>  Welcome to the Show </h1>
```
Comment (stellakunzang) [inline app/views/welcome/index.html.erb:1]: This sets a really fun tone for our site!

**Review by stellakunzang — APPROVED** [review]

File: spec/features/sessions/regular_user_new_spec.rb [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        expect(page).to have_content("Welcome, #{@regular_user.name}")
+        expect(page).to have_link("Log out")
+        expect(page).to have_content("You are now logged in.")
+        expect(page).to_not have_link("Sign up")
+        expect(page).to_not have_link("Sign in")
+    end
+end
+
+# When I visit the login path
+# I see a field to enter my email address and password
```
Comment (sagemlee) [inline spec/features/sessions/regular_user_new_spec.rb:33]: Next time we should probably delete all the commented out stuff.

**Review by sagemlee — APPROVED** [review]

**Pull request merged**